.travis.yml

```diff
@@ -7,9 +7,14 @@ os:
   - osx
 
 julia:
+  - 0.7
```

Comment:
Why 0.7? Everything works fine on 1.0, so I feel like this is just wasting CI.

Reply:
To prove my point: tests fail on 0.7 under Windows... ;-)